.gitlab-ci.yml

```diff
@@ -56,12 +56,13 @@ integration:
     - docker network inspect test >/dev/null 2>&1 || docker network create test
 
     # Run Fencer
+    - CNAME="fencer-"$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 25 ; echo '')
```

Comment:
@mdimjasevic this should have a comment re/ why it was done.

Reply:
Alternatively, I think you can remove it altogether (and `docker network inspect test`), the CI issue was due to old GitLab runner.